lib/plugins/villager.js

```diff
@@ -141,8 +141,11 @@ function inject (bot, { version }) {
       itemCount2 = villager.count(Trade.inputItem2.type, Trade.inputItem2.metadata)
       hasEnoughItem2 = itemCount2 >= Trade.inputItem2.count * count
     }
-    if (!hasEnoughItem1 || !hasEnoughItem2) {
-      throw new Error('Not enough items to trade')
+    if (!hasEnoughItem1) {
+      throw new Error('Not enough item 1 to trade')
+    }
+    if (!hasEnoughItem2) {
+      throw new Error('Not enough item 2 to trade')
```

Comment:
Log a bit more information here. I'm happy to leave this out if it amounts to an unnecessary breaking change.